test/NHSD.BuyingCatalogue.Documents.API.Integration.Tests/Support/IntegrationSetUpFixture.cs

```diff
@@ -1,6 +1,9 @@
 ﻿using System.Threading.Tasks;
+using NUnit.Framework;
 using TechTalk.SpecFlow;
 
+[assembly: Parallelizable(ParallelScope.Fixtures)]
```

Comment:
Might it be worth creating an `AssemblyInfo.cs` file and moving the assembly level attribute to that file? (Most developers will be familiar with the `AssemblyInfo.cs` convention from the .NET Framework.)

(We can't use the .csproj `AssemblyAttribute` element because it only supports `string` parameters.)

Other than `InternalsVisibleTo`, are there any assembly level attributes used in BAPI? If so, what is the convention there?

Reply:
I'm in on the idea of creating Assembly Info property file as it is the cleanest way.

Not that I am aware of, I think BAPI was 'lucky' enough to use only string params and keeps them in the project file.